mdk/src/main/java/com/example/examplemod/ExampleMod.java

```diff
@@ -61,9 +61,9 @@ public class ExampleMod
                 output.accept(EXAMPLE_ITEM.get()); // Add the example item to the tab. For your own tabs, this method is preferred over the event
             }).build());
 
-    public ExampleMod()
+    public ExampleMod(FMLConstructModEvent event)
     {
-        IEventBus modEventBus = FMLJavaModLoadingContext.get().getModEventBus();
+        IEventBus modEventBus = event.getModBus();
```

Comment:
Compile error in the MDK, should be this instead:
```suggestion
        IEventBus modEventBus = event.getModEventBus();
```